sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -846,8 +853,8 @@ class PlanResolutionSuite extends AnalysisTest {
             TableIdentifier(tblName, None), Map("a" -> "1"), false)
           comparePlans(parsed3, expected3)
         } else {
-          val e = intercept[AnalysisException](parseAndResolve(sql3))
-          assert(e.message.contains("DESCRIBE TABLE does not support partition for v2 tables"))
+          // val e = intercept[AnalysisException](parseAndResolve(sql3))
+          // assert(e.message.contains("DESCRIBE TABLE does not support partition for v2 tables"))
```

Comment:
This logic has moved to `DataSourceV2Strategy`.  I will update the tests in `DataSourceV2SQLSuite.scala`.